specification/liftrqumulo/Qumulo.Storage.Management/tspconfig.yaml

```diff
@@ -24,6 +24,13 @@ options:
     flavor: "azure"
     packageDetails:
       name: "@azure/arm-qumulo"
+  "@azure-tools/typespec-csharp":
+    emitter-output-dir: "{project-root}/azure-sdk-for-net/sdk/qumulo/{namespace}/src"
+    package-dir: "Azure.ResourceManager.Qumulo"
+    service-dir: "sdk/qumulo"
```

Comment:
`service-dir` is not emitter option. It is a parameter.. please remove it.

Reply:
sorry. `service-dir` can define under emitter option to override the global one. So we can keep this.